microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
```

Comment:
In toc.yml, change from:
View page resources
to:
View the resource files that make up a webpage

Reply:
done in local copy